lib/openshift/console_external_log_links_console_openshift_io.rb

```diff
@@ -1,9 +1,7 @@
 require 'openshift/cluster_resource'
 
 module BushSlicer
-
   class ConsoleExternalLogLinksConsoleOpenshiftIo < ClusterResource
```

Comment:
@SSshahan I still get error message when I have your fixes, 
```
Given I wait for the "kibana" console_external_log_links_console_openshift_io to appear                                                                    # features/step_definitions/resources.rb:144
      hash not from a ConsoleExternalLogLinksConsoleOpenshiftIo: expected ConsoleExternalLogLinksConsoleOpenshiftIo but was ConsoleExternalLogLink (RuntimeError)
      /home/qitang/github/verification-tests/lib/openshift/resource.rb:140:in `update_from_api_object'
      /home/qitang/github/verification-tests/lib/openshift/resource.rb:82:in `get'
      /home/qitang/github/verification-tests/lib/openshift/resource.rb:43:in `exists?'
      /home/qitang/github/verification-tests/lib/openshift/resource.rb:197:in `block in wait_to_appear'
      /home/qitang/github/verification-tests/lib/base_helper.rb:153:in `wait_for'
      /home/qitang/github/verification-tests/lib/openshift/resource.rb:196:in `wait_to_appear'
      /home/qitang/github/verification-tests/features/step_definitions/resources.rb:149:in `/^(I|admin) waits? for the "(.+?)" (\w+) to appear(?: in the(?: "(.+?)")? project)?(?: up to (\d+) seconds)?$/'
      features/logging/kibana.feature:57:in `Given I wait for the "kibana" console_external_log_links_console_openshift_io to appear'
```

 I think the class should be `ConsoleExternalLogLinkConsoleOpenshiftIo`, and the file name should be `console_external_log_link_console_openshift_io.rb`.
```
$ oc api-resources |grep consoleexternal
consoleexternalloglinks                                console.openshift.io                  false        ConsoleExternalLogLink
```

Reply:
did your case passed locally after apply above change you mentioned? 
oc get crd consoleexternalloglinks.console.openshift.io -o yaml |grep  acceptedNames -A4
  acceptedNames:
    kind: ConsoleExternalLogLink
    listKind: ConsoleExternalLogLinkList
    plural: consoleexternalloglinks
    singular: consoleexternalloglink

Reply:
yes, it passed

> ```
> Given admin waits for the "kibana" console_external_log_link_console_openshift_io to appear                                                                # features/step_definitions/resources.rb:144
>       [08:47:57] INFO> oc get consoleexternalloglinks.console.openshift.io kibana --output=yaml --config=/home/qitang/workdir/wlc-trust-93-qitang/ocp4_admin.kubeconfig
>       [08:47:57] INFO> After 1 iterations and 2 seconds:
>       apiVersion: console.openshift.io/v1
>       kind: ConsoleExternalLogLink
>       metadata:
>         creationTimestamp: "2020-01-06T08:39:28Z"
>         generation: 1
> ```

Reply:
good, I'll update them.

Reply:
I don't know why.

Reply:
@liangxia  ^^ do you know why?

Reply:
```
Given I wait for the "kibana" console_external_log_links_console_openshift_io to appear     # features/step_definitions/resources.rb:144
      hash not from a ConsoleExternalLogLinksConsoleOpenshiftIo: expected ConsoleExternalLogLinksConsoleOpenshiftIo but was ConsoleExternalLogLink (RuntimeError)
```
It is the suffix differences that caused the issue, not the 's'